index.html

```diff
@@ -6,46 +6,32 @@
         <title>ThreeFolds: FOLD creation tool using Three.JS</title>
         <meta name='viewport' content='width=device-width, initial-scale=1'>
         <link rel='stylesheet' type='text/css' media='screen' href='./styles/styles.css'>
-        <link type="text/css" rel="stylesheet" href="main.css">
         <script src="external/three.js"></script>
-        <script src='main.js'></script>
     </head>
     <body>
         <div class='parent flex-parent'>
             <div class='child flex-child'>
                 <canvas id="glCanvas" width="640" height="480"></canvas>
             </div>
             <div class='child flex-child'>
-                <svg id="svg" viewBox="0 0 100 100" width="800" height="600" preserveAspectRatio="none">
-
-
+                <svg id="svg" viewBox="0 0 800 600" width="800" height="600" preserveAspectRatio="none">
```

Comment:
I haven't double-checked how the SVG rendering is handled but you might want this to be square just to match the (square) sheet dimensions.